src/platform/plugins/shared/dashboard/public/dashboard_api/unified_search_manager.ts

```diff
@@ -145,7 +145,6 @@ export function initializeUnifiedSearchManager(
       }
     )
   );
-  controlGroupSubscriptions.add(controlGroupFilters$.subscribe(() => panelsReload$.next()));
```

Comment:
This is what was causing the double fetch. I _think_ this is a holdover from when we stored unified search filters and control group filters differently, and needed some trigger to refresh the page when a control selection changed.

Reply:
You are correct, this `panelsReload$.next()` is carry over from DashboardContainer conversion. Now all filters are passed to children with `filters$` and this is no longer needed.

Since you are moving this, `controlGroupReload$` and `panelsReload$` are no longer needed and can be combine into just `reload$`.